telemetry/wrapper.go

```diff
@@ -11,6 +11,7 @@ const (
 	MetricKeyBeginBlocker = "begin_blocker"
 	MetricKeyEndBlocker   = "end_blocker"
 	MetricLabelNameModule = "module"
+	MetricKeyCommit       = "commit"
```

Comment:
is this being used anywhere?

Reply:
This is a publicly exported key intended for use by the application. It's used [here](https://github.com/dydxprotocol/v4/blob/ff141dd155140b0f5cdbaa5dd2a379fa8a5d3611/x/clob/module.go#L201) in the `clob` module.